src/core/file_format/splash.cpp

```diff
@@ -9,7 +9,7 @@
 #include "splash.h"
 
 bool Splash::Open(const std::filesystem::path& filepath) {
-    ASSERT_MSG(filepath.stem().string() != "png", "Unexpected file format passed");
+    ASSERT_MSG(filepath.extension().string() != ".png", "Unexpected file format passed");
```

Comment:
The assert condition is inverted though.

Reply:
That why it works until now :D.

Reply:
I wonder how many place this kind of assert got inverted.

Reply:
Haha missed that. Thank you.